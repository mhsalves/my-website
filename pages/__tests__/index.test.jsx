import React from 'react';
import renderer from 'react-test-renderer';

import Index from '../index';

describe('Index component', () => {
  it('renders correctly', () => {
    const component = renderer.create(<Index />).toJSON();

    expect(component).toMatchSnapshot();
  });
});
