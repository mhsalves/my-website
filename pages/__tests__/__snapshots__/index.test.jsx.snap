// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index component renders correctly 1`] = `
<div>
  <h1>
    Hello World ! (Home)
  </h1>
  <a
    href="/about"
  >
    Ir para About
  </a>
</div>
`;
